chapters/object_types/frame.txt

```diff
@@ -75,13 +75,21 @@ access the stored pixel information – via
 
 [source,cpp]
 ....
-const void *anariMapFrame(ANARIDevice, ANARIFrame, const char *channel);
+const void* anariMapFrame(ANARIDevice,
+                          ANARIFrame,
+                          const char* channel,
+                          uint32_t *width,
+                          uint32_t *height,
+                          ANARIDataType *type);
 ....
 
 Only channels that have been set as parameters to the frame can be
 mapped, the type of the pixels matches the corresponding parameter
 value.
 
+`width`, `height`, and `type` are output parameters for the application to
```

Comment:
It looks odd to start the sentence lower-case, how about prepending "The arguments `width`..."? Could even merge this into the prev. paragraph line 88.

Reply:
Addressed in latest push, sorry I didn't see this comment!